public/main.js

```diff
@@ -0,0 +1,114 @@
+//ИМПОРТ КЛАССОВ, ПОЧЕМУ ТО, СДЕЛАТЬ НЕ ВЫШЛО. бЫЛО ПЕРЕПРОБЫВАНО ОЧЕНЬ МНОГО. ТАК ЧТО, ИЗВИНИТЕ, НО ОНИ ЗДЕСЬ
+const search_btn = document.getElementById('search_btn');
+const parentBlock = document.getElementById('music_list');
+const libraryStorage = document.getElementById('libraryStorage');
+//ЗАПРОС НА СЕРВЕР
+class HttpRequests {
+    constructor(apiKey, api) {
+        this.requestURL = '';
+        this.apiKey = apiKey;
+        this.api = api;
+    }
+    async sendRequests(url, data) {
+        this.response = await fetch(url, {
+            method: data.method,
+            body: data.body
+        });
+        return await this.response.json();
+    }
+    async searchTrack(track) {
```

Comment:
Тут тоже нет документации

Должно быть что-то вроде
```suggestion
    /**
    * Осуществляет поиск трека
    * {string} track - название трека
    */
    async searchTrack(track) {
```